guides/data/models.md

```diff
@@ -95,7 +95,7 @@ should implement the `number` data type as a `float` or `double` rather than an
 field should only container integers, you should use `integer` (assuming your
 ORM adapter supports it).
 
-The value of the common base set of data types is that it allows addons that
+The value of the common base set of data types is that it allows add-ons that
```

Comment:
"addons" is a term of art here - it's the name for a specific feature of Denali, and is not hyphenated.